dotCMS/src/main/java/com/dotmarketing/portlets/structure/action/EditFieldAction.java

```diff
@@ -552,4 +590,15 @@ public String hostNameUtil(ActionRequest req) {
         return h.getTitle()!=null?h.getTitle():"default";
     }
 
+    /**
+     * Validate if the string is one of the accepted Db boolean values
+     * @param value The boolean string to check
+     * @return true if is a valid boolean, false if not
+     */
+    private boolean isDBBoolean(final String value){
+        return (value.toLowerCase().trim().equals("t") || value.toLowerCase().trim().equals("true")
+                || value.toLowerCase().trim().equals("1") || value.toLowerCase().trim().equals("f")
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move the "1" string literal on the left side of this string comparison. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1132)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move the "f" string literal on the left side of this string comparison. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1132)